_schema.graphql

```diff
@@ -11473,6 +11501,25 @@ type submitPendingOfferPayload {
   clientMutationId: String
 }
 
+type SubtotalAmount {
+  # A formatted price with various currency formatting options.
+  amount(
+    decimal: String = "."
+
+    # Allows control of symbol position (%v = value, %s = symbol)
+    format: String = "%s%v"
+    precision: Int = 0
+    symbol: String
+    thousand: String = ","
+  ): String
+
+  # An amount of money expressed in cents.
+  cents: Float
+
+  # A pre-formatted price.
+  display: String
+}
```

Comment:
Given that this is repeated twice can we do something like `type SuptotalAmount implements Money` ?

https://graphql.org/learn/schema/#interfaces

Reply:
I've also had a chat about this with @dleve123 - this could be a shared `Amount` type rather than defining a type for each amount field. In fact, that's what I did in the first place and later I noticed there is a `money` helper function so I switched to it.

I think there are definitely questions around this or space for improvement. However, that seems like more a foundational change and I'd like a separate thread for that.

Reply:
Here's a PR that introduces a `Price` type which ultimately this should be replaced with. 

In general it's a bad practice to _copy_ a type. I'm going to recommend to platform that we look into tooling to help with this. cc @alloy

Reply:
Tooling to solve what problem specifically?